keep/api/routes/alerts.py

```diff
@@ -689,6 +689,14 @@ async def receive_event(
         raise HTTPException(status_code=400, detail="Malformed event")
     logger.debug("Parsed event raw body", extra={"time": time.time() - t})
 
+    # Format the event using the provider's format_alert method
+    t = time.time()
+    logger.debug("Formatting event using provider's format_alert method")
+    formatted_event = provider_class.format_alert(event, authenticated_entity.tenant_id, provider_type, provider_id)
+    if formatted_event:
+        event = formatted_event
+    logger.debug("Formatted event", extra={"time": time.time() - t})
```

Comment:
I'm a bit confused about this part. Didn't we call format_alert in the other place of the pipeline anyways (in process_event)? :) I'm concerned about performance consideration of this